### ISSUE TITLE
Merging and standardize data

### ISSUE THREAD

**Issue by hoangbros03** [hoangbros03/Translation_Model_10#9]
Currently we have only ~100k training dataset and ~2k test set. Translate and add to current data ASAP.

**Comment by hoangbros03** [comment]
new data coming. I also need to standardize the original provided data files

**Comment by hoangbros03** [comment]
Done. Won't publish into this repo
